### PR TITLE
add: wccom migrate link in nux

### LINKED ISSUE #38420: Migrate Landing Page Link in Onboarding Wizard
## User Story

> As a new WooCommerce user with an existing site elsewhere, I want help migrating my content, so that I can switch to WooCommerce faster.

## Description

In the onboarding wizard, run an experiment during the 7.8 release to direct users to a landing page on WooCommerce.com for assistance migrating a site.

This will be provided by a message that's displayed when the user indicates they have a previous site.

image (9)

### Scenario: User selects a Yes option to "Currently selling elsewhere"

```
Given a user answers the "Currently selling elsewhere" question on the Business Details step of OBW
When <the user selects a Yes option>
Then display the following message: "Need help moving your store? Get free expert advice."
And link "Get free expert advice" to `https://woocommerce.com/migrate/?utm_source=nux&utm_medium=product&utm_campaign=migrate&utm_content=OPTION_KEY`

Examples:
  | The user selects a Yes option                                                                         |
  | Yes, on another platform                                                                                 |
  | Yes, on another platform and in person at physical stores and/or events  |
  | Yes, on another platform and in person at physical stores and/or events  |
 ```

Note: the OPTION_KEY should match the [key property in the code here](https://github.com/woocommerce/woocommerce/blob/81fcbee9d17c560751ee7102f27452981225a257/plugins/woocommerce-admin/client/profile-wizard/steps/business-details/data/selling-venue-options.js#L6-L37).

### PR DESCRIPTION
**Pull request by rjchow**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Adds a link out to woocommerce.com/migrate inside the onboarding wizard when certain options are selected for the business details input.

image

Closes #38420 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install WooCommerce and go through the onboarding wizard until the "Business Details" page. Input values do not matter until then.
2. Observe that when the "Currently selling elsewhere?" dropdown is set to one of "Yes, on another platform", "Yes, I own a different store powered by WooCommerce", or "Yes, on another platform and in person", there is a notice that shows up "Get free expert advice"
3. Clicking on the link should bring up a new window leading to https://woocommerce.com/migrate, with the `&utm_content` parameter corresponding to one of "other", "other-woocommerce", or "brick-mortar-other"

### CODE REVIEW

**Review by chihsuan — DISMISSED** [review]
Thanks for adding the link so quickly! LGTM and tested well. 💯

**Review by adrianduffell — DISMISSED** [review]
LGTM and tested perfectly! I double-checked the links are all good 🚀 

Thank you @rjchow 

Let's just hold on the merge in case @verofasulo wants to take a look first

File: plugins/woocommerce-admin/client/profile-wizard/steps/business-details/flows/selective-bundle/style.scss [hunk 1]
```diff
@@ -20,3 +20,13 @@
 		}
 	}
 }
+
+.woocommerce-profile-wizard__wccom-migration-notice {
+	margin: 0;
+	background: #f0f9fc;
+	height: 50px;
```
Comment (rjchow) [inline plugins/woocommerce-admin/client/profile-wizard/steps/business-details/flows/selective-bundle/style.scss]:
```suggestion
	height: 48px;
```

**Review by adrianduffell — APPROVED** [review]
Latest change is good 🚀

**Pull request merged**